### DISCUSSION TITLE
Remove 3 unnecessary (default) Finder context menus: Compress / Extract / Send to Keka

### DISCUSSION THREAD

**Discussion by jmuheim** [aonez/Keka#1163] [Ideas]
I love the Finder integration:

<img width="682" alt="CleanShot 2022-09-16 at 10 21 22@2x" src="https://user-images.githubusercontent.com/1814983/190591952-af964851-7081-40a7-991c-07028cb7f164.png">

It renders the 3 default operations unnecessary though:

![image](https://user-images.githubusercontent.com/1814983/190592307-72bf46e7-aebc-460e-a3ce-f23264a80f32.png)

Do I miss the option to remove them in Keka's preferences?

**Comment by aonez** [comment]
Those are Keyboard Shortcut Services, you can use them from the Finder using a shortcut:

<img width="580" alt="Screenshot 2022-09-16 at 10 26 38" src="https://user-images.githubusercontent.com/480163/190593092-7cc357ad-ea3f-439e-8ec9-3af00d303f83.png">

Those can be disable in the  Preferences:

<img width="780" alt="Screenshot 2022-09-16 at 10 25 20" src="https://user-images.githubusercontent.com/480163/190593166-70ebbb83-bb5d-41fe-a0a6-a93e82a02ccc.png">

**Reply by aonez** [reply]
Those were the only option before the Finder Extension was available. Just keeping them alive because they can be called with a keyboard shortcut.

**Reply by jmuheim** [reply]
Very good to know! Thank you, I just disabled those.
